entity-framework/core/modeling/query-types.md

```diff
@@ -1,53 +1,49 @@
+> This feature is new in EF Core 2.1. Prior to 3.0 they were known as Query Types
 
-In addition to entity types, an EF Core model can contain _query types_, which can be used to carry out database queries against data that isn't mapped to entity types.
+In addition to normal entity types, an EF Core model can contain _keyless entity types_, which can be used to carry out database queries against data that doesn't contain key values.
 
-## Compare query types to entity types
+## Keyless Entity Types Characteristics
 
-Query types are like entity types in that they:
+Keyless entity types support many of the same mapping capabilities, like inheritance mapping and navigation properties. On relational stores, they can configure the target database objects and columns via fluent API methods or data annotations.
 
-- Can be added to the model either in `OnModelCreating` or via a "set" property on a derived _DbContext_.
-- Support many of the same mapping capabilities, like inheritance mapping and navigation properties. On relational stores, they can configure the target database objects and columns via fluent API methods or data annotations.
+However, they are different from normal entity types in that they:
 
-However, they are different from entity types in that they:
-
-- Do not require a key to be defined.
-- Are never tracked for changes on the _DbContext_ and therefore are never inserted, updated or deleted on the database.
+- Cannot have a key defined.
+- Are never tracked for changes in the _DbContext_ and therefore are never inserted, updated or deleted on the database.
 - Are never discovered by convention.
-- Only support a subset of navigation mapping capabilities - Specifically:
+- Only support a subset of navigation mapping capabilities, specifically:
   - They may never act as the principal end of a relationship.
   - They can only contain reference navigation properties pointing to entities.
-  - Entities cannot contain navigation properties to query types.
-- Are addressed on the _ModelBuilder_ using the `Query` method rather than the `Entity` method.
-- Are mapped on the _DbContext_ through properties of type `DbQuery<T>` rather than `DbSet<T>`
-- Are mapped to database objects using the `ToView` method, rather than `ToTable`.
-- May be mapped to a _defining query_ - A defining query is a secondary query declared in the model that acts a data source for a query type.
+  - Entities cannot contain navigation properties to keyless entity types.
+- Need to be configured with `.HasNoKey()` method call.
+- May be mapped to a _defining query_. A defining query is a query declared in the model that acts as a data source for a keyless entity type.
 
 ## Usage scenarios
 
-Some of the main usage scenarios for query types are:
+Some of the main usage scenarios for keyless entity types are:
 
-- Serving as the return type for ad hoc `FromSql()` queries.
-- Mapping to database views.
+- Serving as the return type for `FromSql()` queries.
```

Comment:
There is no FromSql method anymore :smile:. Rather than adding the two methods:
```suggestion
- Serving as the return type for [raw SQL queries](xref:core/querying/raw-sql).
```